src/transform.ts

```diff
@@ -97,6 +98,57 @@ export interface AggregatedFieldDef {
   as: string;
 }
 
+export interface WindowFieldDef {
+  /**
+   * The operations supported for the window aggregation
```

Comment:
Need to link to the table that lists all supported operations in the docs.

Reply:
Resolved.